README.md

```diff
@@ -122,28 +148,28 @@ In addition to the fields above, each field can have an `action` field which ind
       - create
       - update
 
-Similarly field values can be flexed per action as well
-
-  <pre>Alter the required and note values for update and create
-  - name: id
-    required:
-    - update: 'Yes'
-    - create: 'No'
-    type: id
-    description: The logical id of the resource to update.
-    example: |
-      {
-        "id": "123412"
-      }
-    note:
-    - update: The id value must match the AllergyIntolerance/&lt;id&gt; value.
-    - create: The id field <b>must not</b> be set when performing an update operation.</pre>
+Similarly, field values can be flexed per action as well:
+
+    Alter the required and note values for update and create
+    - name: id
+      required:
+      - update: 'Yes'
+      - create: 'No'
+      type: id
+      description: The logical id of the resource to update.
+      example: |
+        {
+          "id": "123412"
+        }
+      note:
+      - update: The id value must match the AllergyIntolerance/&lt;id&gt; value.
```

Comment:
Note: Looking at the readme on the branch it seems the angle brackets are not being formatted right.

Reply:
nice looks like since i dropped the \<pre\> tags we don't have to use the html entities any longer
https://github.com/cerner/fhir.cerner.com/pull/558/commits/ba15084162fd1b3fd3a969824d5dbfa87c79bffa#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R165